magda-web-client/src/Components/Dataset/Add/Pages/DetailsAndContents/index.tsx

```diff
@@ -268,10 +271,12 @@ export default function DatasetAddAccessAndUsePage(props: Props) {
                     </div>
                     <div className="col-sm-4 question-recent-modify-date">
                         <h4>When was the dataset most recently modified?</h4>
-                        <AlwaysEditor
-                            value={dataset.modified}
-                            onChange={editDataset("modified")}
-                            editor={dateEditor}
+                        <MagdaSingleDatePicker
+                            callback={editDataset("modified")}
+                            date={dataset.modified}
+                            isOutsideRange={(date: moment.Moment) =>
+                                date.isAfter(moment().endOf("day"))
```

Comment:
This puts 'today' out of range too, right? Maybe add a day?
```suggestion
                                date.isAfter(moment().add(1, "day").endOf("day"))
```

Reply:
@sajidanower23 
I deployed a test site:
https://issue-2586.dev.magda.io/
And it seems it works well.
And your code seems read like `any date is outOfRange if it's after end of tomorrow`🤔

Reply:
Oh you are right. I miscalculated in my head :smile: